mflowgen/Tile_MemCore/constraints/common.tcl

```diff
@@ -51,7 +51,7 @@ set_max_transition 0.120 $design_name
 # Constrain INPUTS
 # - make this non-zero to avoid hold buffers on input-registered designs
 set i_delay [expr 0.2 * ${clock_period}]
-set_input_delay -clock ${clock_name} ${i_delay} [all_inputs]
+set_input_delay -clock ${clock_name} ${i_delay} [all_inputs -no_clocks]
```

Comment:
What prompted you to add `-no_clocks` here?

Reply:
It silences a warning from Genus about not being to use `set_input_delay` on ports which have a clock already defined, such as `/designs/Tile_MemCore/ports_in/clk`. Not sure if adding this speeds up Genus but cleaned up the log a bit.

Reply:
ok nice